src/changed-files.js

```diff
@@ -62,6 +62,10 @@ async function changedFiles({
     }
 
     for (let file of _changedFiles) {
+      // if filename has space like `sample file.png`, file will have double quotes "some_directory/sample file.txt".
+      // We need to strip double quotes before pushing to array.
+      file = file.replace(/"/g, '');
```

Comment:
Shouldn't we fix this at the source of changedFiles, which comes from packagesWithChanges -> buildChangeGraph?